### PR TITLE
[release-1.22] Update master to release-1.22 plus update-deps

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

### CODE REVIEW

**Comment by ericvn** [comment]
Might pull out the update_deps part (or at least the ztunnel update) since that is causing the inter-ambient failure right now.

File: manifests/charts/istio-cni/Chart.yaml [hunk 1]
```diff
@@ -9,6 +9,6 @@ keywords:
   - istio-cni
   - istio
 sources:
-  - https://github.com/istio/istio/tree/master/cni
+  - https://github.com/istio/istio/tree/release-1.22/cni
```
Comment (ericvn) [inline manifests/charts/istio-cni/Chart.yaml:12]: I'm not to sure about this being the full source. In the other charts we seem to have:
```
https://github.com/istio/istio
```
which wouldn't need to release updated each release.

Comment (zirain) [reply manifests/charts/istio-cni/Chart.yaml:12]: let's revert this, and fix it in master

File: cni/test/install_k8s_test.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ import (
 
 var (
 	Hub = "gcr.io/istio-release"
-	Tag = "master-latest-daily"
```
Comment (zirain) [inline cni/test/install_k8s_test.go:28]: @ericvn I'm thinking if this's needed?

**Review by jessicegao — APPROVED** [review]
merge first， there is following work on it

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**